form/src/main/kotlin/com/ritense/form/domain/submission/formfield/DataGridField.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2015-2024 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.form.domain.submission.formfield
+
+import com.fasterxml.jackson.core.JsonPointer
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.node.ArrayNode
+import com.fasterxml.jackson.databind.node.ObjectNode
+import com.ritense.document.domain.Document
+import mu.KotlinLogging
+import org.springframework.context.ApplicationEventPublisher
+
+data class DataGridField(
```

Comment:
This seems arbitrary. Wouldn't it be better to apply the normal field logic to any fields that have the collection element of sub fields?

Reply:
Changed